packages/docusaurus-theme-bootstrap/src/theme/CodeBlock/index.js

```diff
@@ -19,14 +19,25 @@ export default ({children, className}) => {
       code={children}
       language={language}>
       {({style, tokens, getLineProps, getTokenProps}) => (
-        <pre className={className} style={{...style, padding: '20px'}}>
-          {tokens.map((line, i) => (
-            <div key={i} {...getLineProps({line, key: i})}>
-              {line.map((token, key) => (
-                <span key={key} {...getTokenProps({token, key})} />
-              ))}
-            </div>
-          ))}
+        <pre
+          style={{
+            margin: '0px',
+            padding: '0px',
+            whiteSpace: 'pre-wrap',
+            wordBreak: 'break-word',
+          }}>
+          <code style={{...style, padding: '20px', display: 'inline-flex'}}>
+            {tokens.map((line, i) => (
+              <div
```

Comment:
The only thing I don't like is that in the code div tag. It doesn't seem to be valid. Try it on:
https://validator.w3.org/nu/#textarea
```
<!DOCTYPE html >
<html lang="en">
<head><title>d</title></head>
<body>

<pre>
   <code>
            
              <div><span>werwer</span>
              </div>
              <div><span>werwer</span>
              </div>
    </code>
</pre>	
</body>
</html>
```
Output:
```
Error: Element div not allowed as child of element code in this context. (Suppressing further errors from this subtree.)
From line 9, column 15; to line 9, column 19
          <div><span>
Contexts in which element div may be used:
Where flow content is expected.
As a child of a dl element.
Content model for element code:
Phrasing content.
```

Reply:
Ohw okay, I will look into it and see what I can do. I initially didn't include it but I noticed the code highlights were not functioning properly. So I added it and it worked as expected.